tailwind.config.js

```diff
@@ -0,0 +1,80 @@
+const colors = require("tailwindcss/colors");
+
+module.exports = {
+  content: [
+    "./integreat_compass/cms/templates/**/*.html",
+    "./integreat_compass/static/src/js/**/*.{js,ts,tsx}",
+  ],
+  theme: {
+    colors: {
+      transparent: "transparent",
+      current: "currentColor",
+      black: colors.black,
+      white: colors.white,
+      gray: colors.slate,
+      redMuted: "#d55854",
+      red: colors.red,
+      orange: colors.orange,
+      yellow: colors.yellow,
+      green: colors.green,
+      blue: colors.blue,
+      violet: colors.violet,
+    },
+    extend: {
+      colors: {
+        water: {
+          50: "#fefeff",
+          100: "#fcfdff",
+          200: "#f8fafe",
+          300: "#f4f7fe",
+          400: "#ecf1fd",
+          500: "#e4ebfc",
+          600: "#cdd4e3",
+          700: "#abb0bd",
+          800: "#898d97",
+          900: "#70737b",
+        },
+        background: {
+          50: "#ebe5e5",
+          900: "#0a001167",
```

Comment:
```suggestion
          900: "#0a0011ab",
```
I think while converting the opacity to hexadecimal something got mixed-up :)

Reply:
Oh, whoops :D